clusterloader2/pkg/framework/framework.go

```diff
@@ -37,18 +38,31 @@ type Framework struct {
 	automanagedNamespaceCount  int
 	clientSets                 *MultiClientSet
 	dynamicClients             *MultiDynamicClient
+	clusterConfig              *config.ClusterConfig
 }
 
+type ClusterType int
+
+const (
+	DefaultCluster ClusterType = iota
```

Comment:
Add some documentation what do you mean by default cluster and by KubemarkRootKube. Also shouldn't it be just "Kubemark" cluster type?

Reply:
Enum removed.